test/pylib/random_tables.py

```diff
@@ -266,8 +266,13 @@ class RandomTables():
     """A list of managed random tables"""
 
     def __init__(self, test_name: str, manager: ManagerClient, keyspace: str,
-                 replication_factor: int, dc_replication_factor: dict[str, int] = None):
+                 replication_factor: int,
+                 dc_replication_factor: dict[str, int] = None,
+                 enable_tablets: bool = None):
```

Comment:
nit: `None` is not a correct `bool`. If you want to use a type hint and support all of `None`, `True`, and `False`, the right type would be `None | bool` or equivalently `Optional[None]`.

Reply:
Done.